include/swift/Sema/ConstraintLocatorPathElts.def

```diff
@@ -96,6 +96,11 @@ SIMPLE_LOCATOR_PATH_ELT(InstanceType)
 /// Also contains the generic parameter type itself.
 CUSTOM_LOCATOR_PATH_ELT(GenericParameter)
 
+/// The former wrappedValue type for a property wrapper.
```

Comment:
I think it's just a `wrappedValue` type for a property wrapped, not `former` since it applies to a type expression of the current wrapper being equated to next one in composition...